packages/sdk/src/cosmwasmclient.spec.ts

```diff
@@ -113,9 +113,7 @@ describe("CosmWasmClient", () => {
     it("works", async () => {
       pendingWithoutCosmos();
       const pen = await Secp256k1Pen.fromMnemonic(faucet.mnemonic);
-      const client = CosmWasmClient.makeWritable(httpUrl, faucet.address, async signBytes => {
-        return encodeSecp256k1Signature(pen.pubkey, await pen.createSignature(signBytes));
-      });
+      const client = CosmWasmClient.makeWritable(httpUrl, faucet.address, signBytes => pen.sign(signBytes));
```

Comment:
Nice.

`..., pen.sign.bind(pen))` would also work but this looks cleaner

Reply:
And definitely simpler than the old way

Reply:
I hardly understand why `this` does not work for plain `pen.sign` and never understood the bind method. I tried to find a linter rule to disallow .bind but did not find any.

Reply:
I think cuz classes aren't really such by syntax sugar over an object with function members. The fact that you can't just pass `pen.sign` and have it call the method of pen is definitely one of the annoying gotchas of modern js